.github/workflows/main.yml

```
@@ -0,0 +1,43 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  Lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: cargo clippy
+
+  Formatting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - run: cargo fmt --verbose --check
+  Tests:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_DB: sea_skipper_axum_example
+          POSTGRES_PASSWORD: changeme
+          POSTGRES_BD: sea_skipper_axum_example
+          SEA_SKIPPER_AXUM_EXAMPLE_DB__DATABASE_HOST: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5433:5433
```

I think the real issue may be here: Postgres listens on `5432` by default, but this is binding the container's port 5433 to the runner on 5433.

( I used `5433` in `axum_example`'s config to avoid conflicting with another DB service I have in localdev on my workstation for another project.)

Let's try binding the container port 5432 to the runner on 5433, I think that will do the trick.
```suggestion
        ports:
          - 5433:5432
```